### ISSUE TITLE
'master' tagged nightly containers not being created for Geveva release of consul

### ISSUE THREAD

**Issue by bnevis-i** [edgexfoundry/docker-edgex-consul#13]
```
$ docker-compose pull
ERROR: manifest for nexus3.edgexfoundry.org:10004/docker-edgex-consul:master not found
```

**Comment by bnevis-i** [comment]
cc @lranjbar 

(Theoretically fixed 5 days ago by PR #11, but doesn't seem to have taken.)

**Comment by bnevis-i** [comment]
Lisa fixed. Closing.
